index.bs

```diff
@@ -5431,54 +5431,72 @@ Appendix A: Retired Terminology</h2>
-	of most recent version of the Process that still used them,
-	or gives the more recent equivalent term when there is one.
+	and it links to the the definition or relevant section
+	in most recent version of the Process that still used them,
+	and gives the more recent equivalent term when there is one.
 
-	Retired concepts:
 	<dl>
-		<dt><dfn noexport id=last-call>Last Call</dfn>
-		<dd><a href="https://www.w3.org/2005/10/Process-20051014/tr.html#last-call">Section 7.4.2 of the 2005 Process</a>
-
-		<dt><dfn noexport id=TeamSubmission>Team Submissions</dfn>
-		<dd><a href="https://www.w3.org/2019/Process-20190301/#TeamSubmission">Section 2.2.1 of the 2019 Process</a>
-
-		<dt><dfn noexport>Edited Recommendation</dfn>
-		<dd><a href="https://www.w3.org/2019/Process-20190301/#maturity-levels">Section 6.1.2 of the 2019 Process</a>
-
-		<dt><dfn noexport id=rec-amended>Amended Recommendation</dfn>
-		<dd><a href="https://www.w3.org/2020/Process-20200915/#rec-amended">Section 6.2.1 of the 2020 Process</a>
-	</dl>
+		<dt>
+			<dfn noexport id=rec-amended>Amended Recommendation</dfn>
+			(retired term)
+		<dd>
+			Last defined in <a href="https://www.w3.org/2020/Process-20200915/#rec-amended">section 6.2.1 of the 2020 Process</a>.
 
-	Renamed concepts:
-	<dl>
-		<dt><dfn noexport id=def-W3CChair>W3C Chair</dfn>
+		<dt>
+			<dfn noexport>Edited Recommendation</dfn>
+			(retired term)
 		<dd>
-			Last used in <a href="https://www.w3.org/2014/Process-20140801/">the 2014 Process</a>,
-			now called [=CEO=].
+			Last defined in <a href="https://www.w3.org/2019/Process-20190301/#maturity-levels">section 6.1.2 of the 2019 Process</a>.
 
-		<dt><dfn noexport id=fdn-member-consortium>Member Consortium</dfn> (plural: Member Consortia)
+		<dt>
+			<dfn noexport id=last-call>Last Call</dfn>
+			(retired term)
 		<dd>
-			Last used in <a href="https://www.w3.org/2021/Process-20211102/">the 2021 Process</a>,
-			now called [=Member Association=].
+			Last defined in <a href="https://www.w3.org/2005/10/Process-20051014/tr.html#last-call">section 7.4.2 of the 2005 Process</a>.
 
-		<dt><dfn noexport id=maturity-levels>Maturity level</dfn>
+		<dt>
+			<dfn noexport id=maturity-levels>Maturity level</dfn>
+			(renamed term)
 		<dd>
-			Last used in <a href="https://www.w3.org/2021/Process-20211102/">the 2021 Process</a>,
+			Last defined <a href="https://www.w3.org/2021/Process-20211102/#maturity-levels">in section 6.3.1 of the 2021 Process</a>,
 			now called [[#maturity-stages|maturity stage]].
 
-		<dt><dfn noexport id=mou>Memorandum of Understanding</dfn> (plural: Memoranda of Understanding, abbreviation: MoU)
+		<dt>
+			<dfn noexport id=fdn-member-consortium>Member Consortium</dfn>, plural: Member Consortia
+			(renamed term)
 		<dd>
-			Last used in <a href="https://www.w3.org/2021/Process-20211102/">the 2021 Process</a>,
+			Last defined <a href="https://www.w3.org/2021/Process-20211102/#fdn-member-consortium">in section 2.1.2.1 of the 2021 Process</a>,
```

Comment:
```suggestion
			Last defined in <a href="https://www.w3.org/2021/Process-20211102/#fdn-member-consortium">section 2.1.2.1 of the 2021 Process</a>,
```

Reply:
Addressed the consistency of "in" being in or outside of the link in https://github.com/w3c/process/pull/898/commits/ef5fa4ebecb7f3670d6082d5e48c0e4e9612afdd